### DISCUSSION TITLE
Any idea how to restore PV from cross aws accounts?

### DISCUSSION THREAD

**Discussion by jagadeesh-kancherla-tfs** [vmware-tanzu/velero#4501] [Community support Q&A]
I have installed velero in one AWS account now we have to migrate everything to other AWS account, is there any way we can do? i tried giving same credentials and restored but PV's are not getting created. Any help is valuable.

**Comment by Satsank** [comment]
Have you verified that the backup-location on source and destination clusters are pointing to the same object storage repository on both clusters? 

velero backup-location get

I've not done this with Velero, but it should be possible. 

Alternatively, something like [CloudCasa](https://cloudcasa.io), which also leverages Velero for CSI functionality, could be used to restore PVs across accounts. If you are restoring EKS clusters, it can also backup EKS metadata of the source cluster and recreate the destination cluster with the same configuration during restore(s).

**Reply by jagadeesh-kancherla-tfs** [reply]
Yes both clusters have same backup-location.
